RNA-seq/nb_cell_line_tximport.md

```diff
@@ -0,0 +1,13 @@
+## Harenza, et al. neuroblastoma cell line data
```

Comment:
Is there something that covers the order of these? Maybe it's the links from the schedule.

Reply:
Yes, was planning to use the schedule.